From 92f478e907094004dbe367db3f6fb42f7fecf5af Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Fri, 4 Oct 2019 21:04:17 +0000
Subject: [PATCH] configure: Add missing dep on libX11

Fixes a link error:

  CCLD     xdg-desktop-portal-gtk
/usr/bin/ld: src/xdg_desktop_portal_gtk-gsd-remote-display-manager.o: undefined reference to symbol 'XQueryExtension'
/usr/bin/ld: /usr/lib/libX11.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 88a9cba..b532412 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,7 +61,7 @@ PKG_CHECK_MODULES(GTK, [xdg-desktop-portal >= 1.0 glib-2.0 >= 2.44 gio-unix-2.0
 AC_SUBST(GTK_CFLAGS)
 AC_SUBST(GTK_LIBS)
 
-PKG_CHECK_MODULES(GTK_X11, gtk+-x11-3.0,
+PKG_CHECK_MODULES(GTK_X11, [gtk+-x11-3.0 x11],
 		  have_gtk_x11=yes, have_gtk_x11=no)
 if test "$have_gtk_x11" = "yes"; then
 	AC_DEFINE(HAVE_GTK_X11, 1, [define if we we have gtk+-x11])
-- 
2.23.0


